app/components/cases/submissions/notifications.js

```diff
@@ -13,16 +14,27 @@ export default class CasesSubmissionsNotificationsComponent extends Component {
   @tracked showNotificationAddressModal = false;
   @tracked emailSettings;
   @tracked isEditing = false;
-
+  @tracked approvalAddresses = new TrackedArray([...this.args.approvalAddresses]);
+  @tracked approvalComment;
+  @tracked notificationAddresses = new TrackedArray([...this.args.notificationAddresses]);
+  @tracked notificationComment;
```

Comment:
these could/should be empty `TrackedArray`. I copied these to the `init` method later.